City-of-Helsinki/hel-fi-drupal-grants
# Hotfix grant handler fields for avustus2

Two things, separate, but outside the stories:
1) Fixing two form fields that were sending false values to Avustus2. This is the part that has to do with code:
 - The form timestamp was using form created -timestamp, not the current time (GrantsHandler.php line 614)
 - The values of the floats in the subventions and other conventions were being sent as integers because of false parsing, this sends them forward as floats (several places in the GrantHandler)

2) composer update, remove old modules, update tunnistamo to tunnistamo 2.0
 - The million other file changes are configuration and language changes.

To test:
pull, composer install, drush cim.
Put the grants handler to debug mode
Go to yleisavustuslomake, fill it with non-integer values for the euro fields. Submit form, see that the values for euro fields retain their floaty nature. See that the correct time is passed to form timestamp field.
